### PR TITLE
build 1.1.1

### PR DESCRIPTION
**Pull request by boldorider4**
* [x] checked [upstream requirements](https://github.com/ctypesgen/ctypesgen/blob/ctypesgen-1.0.2/setup.py)
* [x] forked from [conda-forge](https://github.com/conda-forge/ctypesgen-feedstock)
* [x] checked LICENSE

Notes:
* moved ownership of upstream from `davidjamesc` to `ctypesgen`

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -6,37 +6,40 @@ package:
...
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
-    - python >=2.7
+    - python
     - setuptools
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:22]: Doesn't it also require `setuptools-scm`?

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:22]: Actually, NVM. It' only required on the latest version.

**Review by JeanChristopheMorinPerso — APPROVED** [review]
There is a newer version that we could potentially use. THough I have a feeling maybe this is a dependency for another package that mgith not work with the newest version...

**Review by lorepirri — APPROVED** [review]
LGTM!

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**